### DISCUSSION TITLE
Laravel versioning: 9.49 released, packagist has 9.50.1

### DISCUSSION THREAD

**Discussion by henzeb** [laravel/framework#45906] [Q&A] [answered]
I am just wondering. laravel-news told laravel 9.49 was out. And looking at this repository, that seems the case, however, on packagist we have version 9.50.1, which is the version I just upgraded to.

https://packagist.org/packages/laravel/framework#v9.50.1

I don't know what is going on here, but maybe @taylorotwell or @driesvints can shine a light on this issue?

**Comment by nunomaduro** [comment, marked as answer]
Since 9.49.0, we had to release v9.50.0, followed by a bug fix in v9.50.1. That is why you now see v9.50.1 on Packagist.

**Reply by henzeb** [reply]
I am missing the release notes for 9.50 though. Thanks for the quick response anyway.

**Reply by driesvints** [reply]
Release notes will be posted by the person maintaining that as soon as they're able.

**Comment by brandonburkett** [comment]
Does `changelog.md` receive updates _after_ the actual release?  Currently not seeing 9.50, 9.50.1, or 9.51.

**Reply by driesvints** [reply]
The changelog will get updated as soon as the person that maintaining it will get to it.
